config/config_dev.yaml

```diff
@@ -0,0 +1,71 @@
+log_directory: /report_service/logs/final
+#log_directory: /home/kaipak/datasets/report_service/logs/final
```

Comment:
I recommend removing commented-out lines; they usually lead to suspicion or confusion ("Uh oh was this really needed?", that sort of thing 😁)

If they should stay, they include an _additional_ comment explaining why ("commenting this out but it is useful to put back in during development", that sort of thing 😅)